### ISSUE TITLE
Tailwind Incompatiblity: Textfield/Select outline variant bugs visually

### ISSUE THREAD

**Issue by Dudek-AMS** [hperrin/svelte-material-ui#656]
**Describe the bug**

**To Reproduce**
Include Tailwind to your Project
```html
 <Textfield

                    class="shaped-outlined w-full"
                    variant="outlined"
                    label="something" bind:value={email} type="email" id="input-email" />
```` 

**Screenshots**
![image](https://github.com/hperrin/svelte-material-ui/assets/100142218/2652b077-6b37-4181-9d45-302b558217ea)

**Additional context**
SMUI is missing to set explicetly set border top, left, right to 0

adding this to tailwind css fixed it
```css
.mdc-notched-outline--notched .mdc-notched-outline__notch {
    @apply !border-t-0 !border-l-0 !border-r-0;
}
````

**Comment by sech1p** [comment]
Also for this visible blue shadow...

<img width="249" alt="Image" src="https://github.com/user-attachments/assets/1f972729-f30e-49b3-b111-93795f62d9b9" />

```css
input {
    box-shadow: inset 0px 0px 0px 0px;
}
```

This workaround fixing this.

**Comment by hperrin** [comment]
Yeah, this is an issue with Tailwind, not SMUI. Tailwind shouldn't be polluting native elements.

**Comment by hperrin** [comment]
Thank you for providing the fix. Hopefully anyone else with the issue will see this.
